### ISSUE TITLE
Disconnected pane

### ISSUE THREAD

**Issue by Lapis-Wolf** [minetest/minetest_game#2671]
Hello. I have noticed a bug recently. I don't know if it's the default game or if it has something to do with the mods I have but I have noticed that if I place a glass pane next toa block,it will connect like normal. However, if i place a glass pane on the side of the first one i placed,the two panes will connect to each other and the first one will disconnect from the block, pointing into the other direction. It only started happening recently. The issue is displayed in the image below with the obsidian glass panes. It affects all pane types.
![screenshot_20200506_181346](https://user-images.githubusercontent.com/63688588/81233939-bff42800-8fc5-11ea-8b84-ea1ccfc66e36.png)

**Comment by paramat** [comment]
Please can you explain more clearly, with multiple screenshots?

**Comment by An0n3m0us** [comment]
I believe it's intended to work like that, where the first pane will connect to the second adjacent pane rather than the wall.
https://my.pcloud.com/publink/show?code=XZkCllkZP4gfv523qnQzshShiqBMhbgY3z9k

**Comment by Lapis-Wolf** [comment]
No, it's not supposed to be like that. I tested it in another world without
any mods enabled and the panes behaved like normal (connecting to
everything), so I think it was one of the mods I have. I still don't know
which one though. I'm still testing that.

**Comment by pistachio19zpy** [comment]
wonderful,your thoughts are great except  a little mistake.

**Comment by Lapis-Wolf** [comment]
Well this is just confusing now. I've never seen the panes do that and I've been playing this game for years.

**Comment by paramat** [comment]
Can you reproduce this behaviour using MTG only?

**Comment by paramat** [comment]
> I don't know if it's the default game or if it has something to do with the mods

This bug (if there is one) needs reproducing in MTG with no mods, to remove possible complications.

Also, this needs confirmation that this is the intended behaviour (i am not sure).

**Comment by paramat** [comment]
>  I tested it in another world without
any mods enabled and the panes behaved like normal (connecting to
everything), so I think it was one of the mods I have. I still don't know
which one though. I'm still testing that.

**Comment by Lapis-Wolf** [comment]
Default game....happens by default.

**Comment by paramat** [comment]
Oh sorry, i got confused.
So the issue is whether this is actually intended behaviour or not, as was suggested above.

**Comment by Lapis-Wolf** [comment]
I found that it is intended behavior and apparently,the panes will only
connect to specific blocks and not others. I only know that it will connect
to wood but not glass blocks. Quartz from the 'quartz' mod seems to be
considered a glass block by the game,hence why the panes wouldn't connect
to the quartz house I had built. This system confuses me since I don't know
what all blocks it will connect to and I wish the panes would connect to
all blocks for the sake of simplicity.

-------------------------------------------------

Also,another unrelated issue I want to point out was I noticed in
downloaded maps I tried recently,all the doors turned into unknown nodes.
These were all default doors,not any added in mods. This gets annoying very
quickly, especially with large builds.
